constraints/cv32e40x_core.sdc

```diff
@@ -59,15 +59,19 @@ set in_delay_data_err     [expr $clock_period * 0.80]
 set in_delay_data_exokay  [expr $clock_period * 0.80]
 
 # OBI outputs delays
-set out_delay_instr_req  [expr $clock_period * 0.60]
-set out_delay_instr_addr [expr $clock_period * 0.60]
-
-set out_delay_data_req   [expr $clock_period * 0.60]
-set out_delay_data_we    [expr $clock_period * 0.60]
-set out_delay_data_be    [expr $clock_period * 0.60]
-set out_delay_data_addr  [expr $clock_period * 0.60]
-set out_delay_data_wdata [expr $clock_period * 0.60]
-set out_delay_data_atop  [expr $clock_period * 0.60]
+set out_delay_instr_req     [expr $clock_period * 0.60]
+set out_delay_instr_addr    [expr $clock_period * 0.60]
+set out_delay_instr_memtype [expr $clock_period * 0.60]
+set out_delay_instr_prot    [expr $clock_period * 0.60]
+
+set out_delay_data_req     [expr $clock_period * 0.60]
+set out_delay_data_we      [expr $clock_period * 0.60]
+set out_delay_data_be      [expr $clock_period * 0.60]
+set out_delay_data_addr    [expr $clock_period * 0.60]
+set out_delay_data_wdata   [expr $clock_period * 0.60]
+set out_delay_data_atop    [expr $clock_period * 0.60]
+set out_delay_data_memtype [expr $clock_period * 0.60]
+set out_delay_data_prot    [expr $clock_period * 0.60]
```

Comment:
The nmi_addr_i needs to be added to the early_input_ports group.

fencei_flush_req_o/fencei_flush_ack_i cannot be treated as 'other' (as 'ack' can depend on 'req'). Can you give the 'req' an out_delay of 60% and give 'ack' an in_delay of 80% (don't know if we can meet this yet though, but it would be a good target)

Reply:
Updated.